### PR TITLE
add base images when to build harbor assets

### PR DESCRIPTION
**Pull request by wy65701436**
* add base images when to build harbor assets

Signed-off-by: wang yan <wangyan@vmware.com>

### CODE REVIEW

File: make/photon/Makefile [hunk 1]
```diff
@@ -232,7 +251,7 @@ define _get_binary
 	$(WGET) --timeout 30 --no-check-certificate $1 -O $2
 endef
 
-build: _build_prepare _build_db _build_portal _build_core _build_jobservice _build_log _build_nginx _build_registry _build_registryctl _build_notary _build_clair _build_clair_adapter _build_redis _build_migrator _build_chart_server
+build: set_base_image_version _build_prepare _build_db _build_portal _build_core _build_jobservice _build_log _build_nginx _build_registry _build_registryctl _build_notary _build_clair _build_clair_adapter _build_redis _build_migrator _build_chart_server
```
Comment (bitsf) [inline make/photon/Makefile]: why not pass arg to set harbor_base_image_version

File: make/photon/registryctl/base.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM photon:2.0
+
+Label maintainer="wangyan@vmware.com"
```
Comment (bitsf) [inline make/photon/registryctl/base.Dockerfile]: other images don't have this label

**Comment by heww** [comment]
`Dockerfile.base` looks better than `base.Dockerfile` ?

File: make/photon/Makefile (hunk 1 above)
Comment (wy65701436) [reply make/photon/Makefile]: OK, will change it to build argu

File: make/photon/registryctl/base.Dockerfile (hunk 2 above)
Comment (wy65701436) [reply make/photon/registryctl/base.Dockerfile]: will remove it.

**Pull request merged**